tests/src/testModels.js

```diff
@@ -171,7 +173,7 @@ describe('Model', function() {
         expect(permissions.variables).toEqual('TEST=yes');
     });
 
-    it('loads overrides', function() {
+    it('loads negative overrides', function() {
```

Comment:
How about using the term "negated", instead.

Reply:
Fixed in 602a2dd